include/struct_defs/game_options.h

```diff
@@ -0,0 +1,52 @@
+#ifndef POKEPLATINUM_STRUCT_GAME_OPTIONS_H
```

Comment:
Move this stuff in the same file as game_options.h (do not use struct_defs, that's a temp folder that originated from the automated struct splitting)